pkgs/by-name/ga/galleta/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, stdenv
+, fetchzip
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "galleta";
+  version = "20040505_1";
+
+  src = fetchzip {
+      url = "http://sourceforge.net/projects/fast/files/Galleta/Galleta%20v${finalAttrs.version}/galleta_${finalAttrs.version}.zip";
+      hash = "sha256-tc5XLToyQZutb51ZoBlGWXDpsSqdJ89bjzJwY8kRncA=";
```

Comment:
```suggestion
    url = "mirror://sourceforge/projects/fast/files/Galleta/Galleta%20v${finalAttrs.version}/galleta_${finalAttrs.version}.zip";
    hash = "sha256-tc5XLToyQZutb51ZoBlGWXDpsSqdJ89bjzJwY8kRncA=";
```

Reply:
@SuperSandro2000 I tested this change but when I build the package, I get 404 and it cannot download the source.